Dockerfile

```diff
@@ -52,12 +60,13 @@ COPY ./smt_files/* /smt_files/
 # Solidity example
 COPY src/test/resources/solidity/transaction-reordering.sol /project/example.sol
 
-COPY docker_run_securify /
+# Copy python scripts
+RUN mkdir -p /scripts
+COPY ./scripts/* /scripts/
 
-RUN chmod u+x /docker_run_securify
+COPY docker_run_securify.py /
 
 WORKDIR /
 
-RUN python3 -m sec.scripts.install_solc
-
-CMD ["/docker_run_securify"]
+# ENTRYPOINT allows arguments to be passed (e.g. "--truffle").
+ENTRYPOINT ["python3", "-O", "docker_run_securify.py"]
```

Comment:
I thought that `-O` was relevant only to heavily optimize Python code, but here the time spent with Python should be negligible (compared with Securify itself), why is it used here?

Reply:
My reasoning was why not? It would also allow to use `assert`s and `debug` flags to check code behaviour when not using docker that would be ignored inside the docker.

Reply:
If we use Python assertions that make some bugs trivial to identify when they are triggered, but are somewhat silenced in some cases, it will be harder to debug, and I don't see the benefit.

Reply:
Ok I'll remove the optimisation flag then